test/InterversionTests/SmugglerTests.cs

```diff
@@ -21,366 +23,25 @@ namespace InterversionTests
-                    Assert.Equal(0, record.SinkPullReplications.Count);
-                    Assert.Equal(0, record.HubPullReplications.Count);
-                    Assert.Equal(0, record.RavenEtls.Count);
-                    Assert.Equal(0, record.SqlEtls.Count);
-                    Assert.Equal(0, record.PeriodicBackups.Count);
-                }
-            }
-            finally
-            {
-                File.Delete(file);
-            }
+            Non,
+            Export,
+            Import
         }
-
-        [Fact]
-        public async Task CanExportAndImportCurrentClientWithServer41()
+        
+        public SmugglerTests(ITestOutputHelper output) : base(output)
         {
-            var file = GetTempFileName();
-            try
-            {
-                using (var store41 = await GetDocumentStoreAsync("4.1.4"))
-                {
-                    using (var session = store41.OpenSession())
-                    {
-                        for (var i = 0; i < 5; i++)
-                        {
-                            session.Store(new User { Name = "raven" + i });
-                        }
-                        session.SaveChanges();
-                    }
-                    var operation = await store41.Smuggler.ExportAsync(new DatabaseSmugglerExportOptions()
-                    {
-                        OperateOnDatabaseRecordTypes = DatabaseRecordItemType.PeriodicBackups,
-                        OperateOnTypes = DatabaseItemType.DatabaseRecord
-                    }, file);
-                    await operation.WaitForCompletionAsync(TimeSpan.FromMinutes(1));
-
-                    var stats = await store41.Maintenance.SendAsync(new GetStatisticsOperation());
-
-                    long countOfDocuments = stats.CountOfDocuments;
-                    var options = new DatabaseSmugglerImportOptions()
-                    {
-                        OperateOnDatabaseRecordTypes = DatabaseRecordItemType.PeriodicBackups,
-                        OperateOnTypes = DatabaseItemType.DatabaseRecord
-                    };
-                    options.SkipRevisionCreation = true;
-
-                    operation = await store41.Smuggler.ImportAsync(options, file);
-                    await operation.WaitForCompletionAsync(TimeSpan.FromMinutes(1));
-
-                    stats = await store41.Maintenance.SendAsync(new GetStatisticsOperation());
-
-                    Assert.Equal(countOfDocuments, stats.CountOfDocuments);
-                }
-            }
-            finally
-            {
-                File.Delete(file);
-            }
         }
 
-        [Fact]
-        public async Task CanExportFrom42AndImportToCurrent()
+        [Theory]
+        [InlineData(ExcludeOn.Non)]
+        [InlineData(ExcludeOn.Export)]
+        [InlineData(ExcludeOn.Import)]
+        public async Task CanExportFrom42AndImportTo5(ExcludeOn excludeOn)
```

Comment:
It is not '5'. It is 'Current'. E.g. what will happen if we will move to v6? Same for other names.